egress/tests/test_numeric.py

```diff
@@ -18,22 +24,41 @@ def setUpModule():
     connection = connect()
     connection._autocommit = True
     with connection.cursor() as cursor:
-        create_db(cursor)
+        create_db(cursor, db_name)
 
 
 def tearDownModule():
     '''Drop the database and close the connection'''
     connection = connect()
     connection._autocommit = True
     with connection.cursor() as cursor:
-        drop_db(cursor)
-
-    connection.close()
+        drop_db(cursor, db_name)
 
 
 class TestNumericTypes(unittest.TestCase):
     '''
     Tests for postgres numeric types
     '''
-    def test_numeric(self):
-        self.skipTest('TBD')
+    def execute_operation_on_db(self, operation):
+        connection = connect(dbname=db_name)
+        connection._autocommit = True
+        with connection.cursor() as cursor:
+            cursor.execute(operation)
+            return cursor.rowcount
+
+    def setUp(self):
```

Comment:
Do we want to create/destroy this table _every_ test function?
Why can't it live in setUpClass / tearDownClass ?